example/src/constants.js

```diff
@@ -1,13 +1,4 @@
-import {
-  Copy,
```

Comment:
<img width="1440" alt="Screenshot 2022-02-14 at 11 55 49 AM" src="https://user-images.githubusercontent.com/48869249/153811294-0a7d04c4-282c-4629-a1aa-a37e2e0efa00.png">

This file is being used in Storybook documentation. Let’s restore the icons.